lib/spack/spack/solver/asp.py

```diff
@@ -4332,8 +4550,6 @@ def __init__(self, provided, conflicts):
 
         super().__init__(msg)
 
-        self.provided = provided
```

Comment:
Is there a reason to remove this? I think it's better to keep it so that code catching the exception can access it if needed.

Reply:
I don't remember doing this, so I assume it was a rebase/merge artifact. I'll add it back.